src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculetion {
+
+    private double baseB;
+    private double baseA;
+    private double height;
+
+    public IsoscelesTrapezoid(
+            double height,
+            double baseA,
+            double baseB,
+            String color,
+            String name
+    ) {
+        this.height = height;
+        this.baseA = baseA;
+        this.baseB = baseB;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public double getBaseA() {
+        return baseA;
+    }
+
+    public double getBaseB() {
+        return baseB;
+    }
+
+    @Override
+    public double getArea() {
+        return (getBaseA() + getBaseB()) * getHeight() * COEFFICIENT;

```

Comment:
Avoid using getters inside the same class when you can use the fields directly. You can use `this.baseA`, `this.baseB`, and `this.height` instead of the getter methods.